test/test_helper.rb

```diff
@@ -8,6 +8,8 @@
 require 'minitest/autorun'
 require 'minitest/spec'
 
+require 'byebug'
```

Comment:
is this necessary? or just left behind from debugging?

Reply:
This allows one to insert ‘byebug’ in specs when writing them to debug. Without this, what’s the point in specifying the dependency? It’s there to make one’s life easy when working on specs, no?

Reply:
Ah I didn't realize it was already in the gemspec as a dependency. Thanks!